instrumentation/kamon-jdbc/src/test/scala/kamon/instrumentation/jdbc/StatementInstrumentationSpec.scala

```diff
@@ -89,7 +89,7 @@ class StatementInstrumentationSpec extends WordSpec with Matchers with Eventuall
           statement.execute()
           validateNextRow(statement.getResultSet, valueNr = 1, valueName = "foo")
 
-          eventually(timeout(scaled(5 seconds))) {
+          eventually(timeout(scaled(5 seconds)), interval(200 millis)) {
```

Comment:
This test seemed really flaky, pr build failed a few times because of it. Seems OK with this change, although could just be coincidence

Reply:
Thanks!